indico/modules/events/contributions/controllers/management.py

```diff
@@ -175,8 +178,10 @@ class RHCreateContribution(RHManageContributionsBase):
     def _process(self):
         inherited_location = self.event.location_data
         inherited_location['inheriting'] = True
+        default_duration = TimedeltaConverter.to_python(event_core_settings.get(self.event, 'contrib_duration'))
```

Comment:
Don't use the converter here; specify it in the EventSettingsProxy - it will automatically convert on get/set.